src/main/java/com/blazemeter/taurus/junit/generator/Worker.java

```diff
@@ -0,0 +1,101 @@
+package com.blazemeter.taurus.junit.generator;
+
+import com.blazemeter.taurus.junit.CustomRunner;
+import com.blazemeter.taurus.junit.JUnitRunner;
+import com.blazemeter.taurus.junit.reporting.TaurusReporter;
+import com.blazemeter.taurus.junit4.JUnit4Runner;
+import com.blazemeter.taurus.junit5.JUnit5Runner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.blazemeter.taurus.junit.CustomRunner.*;
+
+public class Worker extends Thread {
```

Comment:
Is this a non-daemon thread? What if thread will hang because of infinite loop in test case? Supervisor has to have bulletproof way to interrupt worker threads.